en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Supported modules
+description: Modules which are commercially supported by Silverstripe
+---
+
+# Commercially supported modules
+
+Silverstripe CMS ships modules that receive commercial support. Commercially supported modules receive regular updates to work with the latest Silverstripe CMS release. Their APIs conform to [semantic versioning](https://semver.org/). They are covered by:
+- our [security release process](managing_security_issues) and 
+- our [major release policy](major_release_policy).
+
+These modules will be supported for the lifetime of Silverstripe CMS 4 in the provided major versions.
+
+[notice]
+This list is for Silverstripe CMS 4. For Silverstripe CMS 5 supported modules, see the [CMS 5 version of this page](/en/5/project_governance/supported_modules/).
+[/notice]
```

Comment:
There's already a banner mechanism for this.
```suggestion
```

Reply:
There's an AC for this :-) `The CMS 4 and CMS 5 supported modules doc pages link to each other and **make explicit that module support will vary from version to version**.`